backend/editor/scheduler.py

```diff
@@ -0,0 +1,35 @@
+import contextlib
+import logging
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+
+from .controllers.project_controller import delete_project, get_projects_by_status
+from .github_functions import GithubOperations
+from .graph_db import TransactionCtx
+from .models.project_models import ProjectStatus
+
+log = logging.getLogger(__name__)
+
+
+async def delete_merged_projects():
+    async with TransactionCtx():
+        exported_projects = await get_projects_by_status(ProjectStatus.EXPORTED)
+        for project in exported_projects:
+            pr_number = project.github_pr_url and project.github_pr_url.split("/")[-1]
```

Comment:
nit: The small optimization that's really not needed but exists :grin::
```suggestion
            pr_number = project.github_pr_url and project.github_pr_url.rsplit("/", 1)[-1]
```